docs/configuration/arguments.md

```diff
@@ -183,6 +183,12 @@ prefix these flags with `distributor.ha-tracker.`
    The timeout for the etcd connection.
 - `etcd.max-retries`
    The maximum number of retries to do for failed ops.
+- `etcd.cert-file`
+   The etcd cert file directory location.
```

Comment:
Would you mind updating these descriptions with the new descriptions we put in the code, please?